lib/internal/process.js

```diff
@@ -3,6 +3,11 @@
 const errors = require('internal/errors');
 const util = require('util');
 const constants = process.binding('constants').os.signals;
+const {
+  getHiddenValue,
+  setHiddenValue,
+  is_exiting_private_symbol: kIsExiting
```

Comment:
`kIsExitingSymbol`?  `kIsExiting` suggests it's a boolean.

Reply:
Aren't we using `kAnything` to signify symbols where, I assume, `k === key`? At the very least, http2, tls & url already do it that way.

Reply:
@apapirovski `k` is for **k**onstants ;)